app/src/controllers/project/selectors.js

```diff
@@ -55,7 +57,15 @@ export const projectNameSelector = (state) => projectInfoSelector(state).project
 export const organizationNameSelector = createSelector(
   assignedOrganizationsSelector,
   projectInfoSelector,
-  (assignedOrganizations, { organizationId }) => {
+  userAccountRoleSelector,
+  organizationsListSelector,
+  (assignedOrganizations, { organizationId }, accountRole, organizations) => {
+    const isAdmin = accountRole === ADMINISTRATOR;
+
+    if (isAdmin) {
+      return organizations.find(({ id }) => id === organizationId)?.name;
+    }
+
```

Comment:
Can we select the current org name from `activeOrganization`?

Reply:
Done.